source-airtable/__main__.py

```diff
@@ -0,0 +1,7 @@
+from python import shim_airbyte_cdk
+from source_airtable import SourceAirtable
+
+shim_airbyte_cdk.CaptureShim(
+    delegate=SourceAirtable(),
+    oauth2=None
```

Comment:
nit: there's a value for this spec already in the DB that we can pluck out and place here

Reply:
Done